mmedit/datasets/sr_landmark_dataset.py

```diff
@@ -0,0 +1,63 @@
+import os.path as osp
+
+import numpy as np
+
+from .base_sr_dataset import BaseSRDataset
+from .registry import DATASETS
+
+
+@DATASETS.register_module()
+class SRLandmarkDataset(BaseSRDataset):
+    """General image and landmark dataset with an annotation file for image
+    restoration.
+
+    The dataset loads gt (Ground-Truth) image, shape of image, face box, and
```

Comment:
Is it for general purposes or for faces only?

Reply:
Faces only

Reply:
So would it be better to change it to `SRFacicalLandmarkDataset`? Also, the "general" in line 11 should be changed?